theme/pxt.less

```diff
@@ -9,6 +9,8 @@
 
 @import 'accessibility';
 @import 'highcontrast';
+@import 'fieldeditors';
+@import 'serial';
```

Comment:
Pull latest master. Field editors shouldn't be here but in blockly-core.less

Nothing should go after accessibility. so have serial before that.

Reply:
Fixed